microsoft-edge/web-platform/user-agent-guidance.md

```diff
@@ -211,6 +211,14 @@ On desktop operating systems, Microsoft Edge is commonly identified by the `Edg`
 | iPhone/iPad | `EdgiOS` |
 | Android (Mobile/tablet) | `EdgA`|
 
+### Microsoft Edge Legacy identifier
+
+The Microsoft Edge Legacy browser does not support User Agent Client Hints. It can only be differentiated from Chromium-based Microsoft Edge using the User-Agent string.
+
+| Platform | Identifier token |

```

Comment:
Microsoft Edge Legacy was only ever available on 1 platform, Windows 10. Having this section as a subheader in the platforms section and a table that mentions platform still doesn't quite seem right to me.

What if we add a `##` section at the end titled something like `Microsoft Edge Legacy`. In there, we could comment on things like:
- The old browser is no longer supported (https://techcommunity.microsoft.com/t5/microsoft-365-blog/new-microsoft-edge-to-replace-microsoft-edge-legacy-with-april-s/ba-p/2114224)
- It was only available on Windows 10
- Of everything mentioned above, the only supported browser-specific mechanism that the old browser supports is User Agent strings
- The user agent string for Edge Legacy includes an `Edge` token
- A full example of an Edge Legacy User Agent string: `Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/70.0.3538.102 Safari/537.36 Edge/18.19582`

Reply:
No problem, take 2 coming up

Reply:
Let me know if this version works better.